logstash-filter-aggregate.gemspec

```diff
@@ -6,7 +6,7 @@ Gem::Specification.new do |s|
   s.description = "This gem is a logstash plugin required to be installed on top of the Logstash core pipeline using $LS_HOME/bin/plugin install gemname. This gem is not a stand-alone program"
   s.authors = ["Elastic", "Fabien Baligand"]
   s.email = 'info@elastic.co'
-  s.homepage = "http://www.elastic.co/guide/en/logstash/current/index.html"
+  s.homepage = "https://github.com/logstash-plugins/logstash-filter-aggregate"
```

Comment:
@purbon 
I understand the relevance to have a centralized documentation.
But for aggregate plugin, one thing is currently missing : There is no entry "aggregate" in the filter plugins list (https://www.elastic.co/guide/en/logstash/current/filter-plugins.html).
Can it be added ?
In this pull request, I have enhanced asciidoc documentation for that.